libs/newsletter-workflow/src/lib/steps/editDraftNewsletter.ts

```diff
@@ -0,0 +1,32 @@
+import type { WizardStepLayout } from '@newsletters-nx/state-machine';
+import { executeCreate } from '../executeCreate';
+
+export const editDraftNewsletterLayout: WizardStepLayout = {
+	staticMarkdown: `# Edit a draft newsletter
+
+This wizard allows you to edit an existing draft newsletter, guiding you through the process of entering the data needed to create and launch a new newsletter using email-rendering.
+
+The first step is to choose a name for your newsletter.
+
+For example:
+  "Down to Earth"
+
+`,
+	buttons: {
+		cancel: {
+			buttonType: 'RED',
+			label: 'Cancel',
+			stepToMoveTo: 'cancel',
+		},
+		next: {
+			buttonType: 'GREEN',
+			label: 'Next',
+			stepToMoveTo: 'identityName',
+			onBeforeStepChangeValidate: (stepData): string | undefined => {
+				const name = stepData.formData ? stepData.formData['name'] : undefined;
+				return name ? undefined : 'NO NAME PROVIDED';
+			},
+			executeStep: executeCreate,
```

Comment:
maybe we should rename `executeCreate` if it is also used to start editing and existing draft

Reply:
It should be calling executeModify at this stage, as executeCreate is only needed for a new newsletter - thanks!